libs/blocks/bulk/bulk.js

```diff
@@ -67,117 +65,117 @@ function prettyDate() {
   `;
 }
 
-function StatusRow({ row }) {
-  return html`
-    <tr class="bulk-status-row">
-        <td class="bulk-status-url">${row.status.preview && row.url}</td>
-        <td class="bulk-status-preview">${row.status.preview && row.status.preview}</td>
-        <td class="bulk-status-url">${row.status.publish && row.url}</td>
-        <td class="bulk-status-publish">${row.status.publish && row.status.publish}</td>
-    </tr>
+function bulkPublishStatus(row) {
+  const status = row.status.publish !== 200
+    ? `Error  - Status: ${row.status.publish}`
+    : '';
+  return row.status.publish !== 200 && row.status.publish !== undefined  && html`
+    <span class=page-status>${status}</span>
+  `;
+}
+
+function bulkPreviewStatus(row) {
+  const status = row.status.preview !== 200
+    ? `Error - Status: ${row.status.preview}`
+    : '';
+  return row.status.preview !== 200 && row.status.preview !== undefined && html`
+    <span class=page-status>${status}</span>
   `;
 }
 
-function StatusTitle({
-  bulkTriggered, submittedAction, copyResults, showHideUrls, bulkInputToggleElt, urlNumber,
-}) {
+function StatusTitle({ bulkTriggered, submittedAction, urlNumber }) {
   const name = getActionName(submittedAction, true);
   const URLS = (urlNumber > 1) ? 'URLS' : 'URL';
   return bulkTriggered && html`
-      <div class="bulk-status-head">
-          <div class="bulk-status-head-title">
-              STATUS ${name} ${urlNumber} ${URLS}
-          </div>
-          <div>
-              <button class="bulk-status-head-copy" onclick=${copyResults} title="Copy results"></button>
```

Comment:
@jckautzmann We decided to remove copy results button because it wasn't requested and questioned it getting used. Also, one less thing to maintain.